medical_physician/models/medical_physician.py

```diff
@@ -38,6 +38,28 @@ class MedicalPhysician(models.Model):
         comodel_name='medical.physician.schedule.template',
         inverse_name='physician_id',
     )
+    medical_center_ids = fields.Many2many(
+        string='Primary Medical Center',
+        comodel_name='medical.center',
+        help="The physician's medical centers.",
+        compute='_compute_medical_center_ids',
+        store=True,
+        readonly=False,
+    )
+
+    @api.multi
+    @api.depends('parent_id')
+    def _compute_medical_center_ids(self):
+        for record in self:
+            center = self.env['medical.center'].search([
+                ('partner_id', '=', record.parent_id.id),
+            ], limit=1)
+
+            if not center:
+                continue
+
+            if center not in record.medical_center_ids:
+                record.medical_center_ids += center
```

Comment:
Oh actually no I think this bit is working. However I think what happens is when the compute function is triggered, the field is completely emptied of the records before running code in the compute method. I'm not exactly sure what to do in that case.